projects/1.20/assets/create-connected/create-connected/lang/zh_cn.json

```
@@ -0,0 +1,244 @@
+{
+    "advancement.create_connected.brass_gearbox": "精心安排",
+    "advancement.create_connected.brass_gearbox.desc": "放下一个黄铜十字齿轮箱",
+    "advancement.create_connected.control_chip": "精密制造",
+    "advancement.create_connected.control_chip.desc": "组装一个控制芯片",
+    "advancement.create_connected.overpowered_brake_0": "压倒性的动力",
+    "advancement.create_connected.overpowered_brake_0.desc": "使用被红石激活的制动器时仍然保持动力网络高速运行\n§7（隐藏进度）",
```

感觉这个进度的译名与其他的风格不太一致啊
描述也有点奇怪

自己起了个名字，但也未见得很好（
```suggestion
    "advancement.create_connected.overpowered_brake_0": "力大砖飞",
    "advancement.create_connected.overpowered_brake_0.desc": "克服制动器的制动，保持动力网络高速运行\n§7（隐藏进度）",
```

---

名称我觉得力大砖飞确实挺好，描述要不要改成”在制动器激活的情况下，仍保持动力网络高速运行。\n§7（隐藏进度）“

---

可以